health/health.d/apcupsd.conf

```diff
@@ -60,3 +60,66 @@ component: UPS device
     delay: up 0 down 15m multiplier 1.5 max 1h
      info: APC UPS self-test failed due to insufficient battery capacity or due to overload.
        to: sitemgr
+
+#Send out a warning when STATUS code is ONBATT,OVERLOAD,LOWBATT,REPLACEBATT,NOBATT,COMMLOST
+#https://man.archlinux.org/man/apcaccess.8.en#:~:text=apcupsd%20was%20started-,STATUS,-%3A%20UPS%20status.%20One
+
+ template: apcupsd_status_onbatt
+       on: apcupsd.status
+   lookup: max -1s unaligned match-names of ONBATT
+    units: status
+    every: 10s
+     warn: $this == 1
+    delay: up 1m down 15m multiplier 1.5 max 1h
+     info: APC UPS has switched to battery power because the input power has failed
+       to: sitemgr
+
+ template: apcupsd_status_overload
+       on: apcupsd.status
+   lookup: max -1s unaligned match-names of OVERLOAD
+    units: status
+    every: 10s
+     warn: $this == 1
+    delay: up 0 down 15m multiplier 1.5 max 1h
+     info: APC UPS is overloaded and cannot supply enough power to the load
+       to: sitemgr
+
+ template: apcupsd_status_lowbatt
+       on: apcupsd.status
+   lookup: max -1s unaligned match-names of LOWBATT
+    units: status
+    every: 10s
+     warn: $this == 1
+    delay: up 0 down 15m multiplier 1.5 max 1h
+     info: APC UPS battery is low and needs to be recharged
+       to: sitemgr
+
+ template: apcupsd_status_replacebatt
+       on: apcupsd.status
+   lookup: max -1s unaligned match-names of REPLACEBATT
+    units: status
+    every: 10s
+     warn: $this == 1
+    delay: up 0 down 15m multiplier 1.5 max 1h
+     info: APC UPS battery has reached the end of its lifespan and needs to be replaced
+       to: sitemgr
+
+ template: apcupsd_status_nobatt
+       on: apcupsd.status
+   lookup: max -1s unaligned match-names of NOBATT
+    units: status
+    every: 10s
+     warn: $this == 1
+    delay: up 0 down 15m multiplier 1.5 max 1h
+     info: APC UPS has no battery
+       to: sitemgr
+
+ template: apcupsd_status_commlost
```

Comment:
I think we will never see COMMLOST. Because if the connection is lost, we will not be able to poll the UPS. Right, @thomasbeaudry?

Reply:
Hello @thomasbeaudry,

Please, take a look in this @ilyam8's question.

Best regards!

Reply:
@thiagoftsm I will do a separate PR if needed.